tests/ArmTemplates.Tests/Resources/GraphqlSchema/spacexSchema.gql

```diff
@@ -0,0 +1,1116 @@
+schema {
+  query: Query
+  mutation: Mutation
+  subscription: Subscription
+}
```

Comment:
moved the spacexSchema.gql to open api spec folder to be consistent

Reply:
May I clarify from where the example is generated? 
Is it some generic example for the graphql?
I just thinking from test perspective: 
  Test verifies that generated template has the apischema and equals to the value which client returned. 
In such a case will it be more convenient to has simple example like hello world for instance? 
The original comment was not referred to the location of file :)

Reply:
_May I clarify from where the example is generated?_ -----> This is  GraphQLSchema file of graphql api which we are using for development and testing purposes


_Test verifies that generated template has the apischema and equals to the value which client returned.
In such a case will it be more convenient to has simple example like hello world for instance?
The original comment was not referred to the location of file :)_ ------> Actually I need to verify schema document value generated by apitemplate is matching schema file in resources folder, because this was the actual bug where additional back slashes were being added to schema document in template file